client/src/components/Draftail/decorators/Link.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Icon from '../../Icon/Icon';
+
+import TooltipEntity from '../decorators/TooltipEntity';
+
+const getEmailAddress = mailto => mailto.replace('mailto:', '').split('?')[0];
```

Comment:
This was because links copy-pasted from rich text (eg. Apple Pages) sometimes could have a `?subject=` query param. I realise this isn't supported by Wagtail's link chooser though, so I'll remove this and filter out `mailto:` links on paste instead.